vertical-pod-autoscaler/pkg/recommender/logic/recommender.go

```diff
@@ -31,9 +32,11 @@ var (
 	targetCPUPercentile        = flag.Float64("target-cpu-percentile", 0.9, "CPU usage percentile that will be used as a base for CPU target recommendation. Doesn't affect CPU lower bound, CPU upper bound nor memory recommendations.")
 	lowerBoundCPUPercentile    = flag.Float64("recommendation-lower-bound-cpu-percentile", 0.5, `CPU usage percentile that will be used for the lower bound on CPU recommendation.`)
 	upperBoundCPUPercentile    = flag.Float64("recommendation-upper-bound-cpu-percentile", 0.95, `CPU usage percentile that will be used for the upper bound on CPU recommendation.`)
+	cpuConfidenceInterval      = flag.Duration("cpu-confidence-interval", time.Hour*24, "The time interval used for computing the confidence multiplier for the CPU target recommendation. Default: 24h")
```

Comment:
A bit of a nit:
```suggestion
	confidenceIntervalCPU      = flag.Duration("confidence-interval-cpu", time.Hour*24, "The time interval used for computing the confidence multiplier for the CPU target recommendation. Default: 24h")
```

This is a nit, since I don't think we have a standard set, but a lot of the other parameters don't have the resource as the first part of the parameter (I know this isn't true everywhere).

I like having the resource last, that way the parameters are always next to other when ordered alphabetically:
```
--confidence-interval-cpu
--confidence-interval-memory
```

Again, it's a big nit and a personal preference. We also don't have a good standard here.

Reply:
Done. Thanks for the suggestion!